File: backend/model/Listing.py
from app import db,ma

class Listing(db.Model):
    listing_id = db.Column(db.Integer, primary_key = True)
    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable = False)
    usd_amount = db.Column(db.Float,nullable = False)
    rate = db.Column(db.Float, nullable = False)
    usd_to_lbp = db.Column(db.Boolean, nullable = False)

class ListingSchema(ma.Schema):
    class Meta:
        fields = ("listing_id", "user_id", "usd_amount","rate","usd_to_lbp")
        model = Listing

File: android/app/src/main/java/com/kfo00/currencyexchange/api/model/Listing.kt
package com.kfo00.currencyexchange.api.model

import com.google.gson.annotations.SerializedName

class Listing {


    @SerializedName("listing_id")
    var listing_id :Int? = null

    @SerializedName("usd_amount")
    var usd_amount: Float? = null

    @SerializedName("rate")
    var rate: Float? = null

    @SerializedName("usd_to_lbp")
    var usd_to_lbp: Boolean?  = null

    @SerializedName("user_id")
    var user_name : String? = null


}